Source/App/HeadlessMain.cpp

```
@@ -512,7 +612,11 @@ static JSON makeReportJson(const Scenario& sc, const RunMetrics& m) {
 
     JSON r = JSON::object();
     r.set("name", JSON(sc.name));
-    r.set("type", JSON(sc.type == ScenarioType::Project ? "project" : sc.type == ScenarioType::Tone ? "tone" : "stress"));
+    const char* typeStr = "project";
+    if (sc.type == ScenarioType::RoundTrip) typeStr = "roundtrip";
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)